carbonmark-data/components/ChangeLanguageButton/index.tsx

```diff
@@ -2,25 +2,38 @@
 
 import { t } from "@lingui/macro";
 import Language from "@mui/icons-material/Language";
+import { Button } from "@mui/material";
 import Tippy from "@tippyjs/react";
-import { FC } from "react";
+import { FC, useState } from "react";
 import * as styles from "./styles";
 
 export const ChangeLanguageButton: FC = () => {
+  const [showTooltip, setShowTooltip] = useState(false);
+
   return (
     <Tippy
       className={styles.tooltip}
-      content={<></>}
+      content={
+        <div>
+          <div>
+            <Button onClick={() => setShowTooltip(false)}>English</Button>
+          </div>
+          <div>
+            <Button onClick={() => setShowTooltip(false)}>Deutsch</Button>
+          </div>
+        </div>
+      }
```

Comment:
We are implementing the real list of languages later then (I think it is better because I have i18n implementaion fixes coming in another PR)?

Reply:
Yeah, since this PR was mostly about styling and layouting I wanted to add a dummy language switcher. We should do the real implementation separately 👍